tests/web/Glpi/Inventory/Request.php

```diff
@@ -40,7 +40,7 @@
 use GuzzleHttp\Exception\RequestException;
 use GuzzleHttp\Psr7\Response;
 
-class Request extends \GLPITestCase
+class Request extends \DBTestCase
```

Comment:
```suggestion
class Request extends \GLPITestCase
```

revert to `GLPITestCase`